src/main/java/core/basesyntax/MainApp.java

```diff
@@ -2,4 +2,16 @@
 
 public class MainApp {
 
+    public static void main(String[] args) {
+        Machine truck = new Truck();
+        Machine bulldozer = new Bulldozer();
+        Machine excavator = new Excavator();
+
+        Machine[] machines = {truck, bulldozer, excavator};
+
+        for (Machine machine : machines) {
+            machine.doWork();
+            machine.stopWork();
+        }
+    }
```

Comment:
Please remove redundant empty lines after the class declaration and before the closing brace of the class.